dhis2/dhis2-core
# feat: add login app to default bundled apps

## Summary

* Adds the new login-app to the default bundled apps list.
* Sets the new login app as the default login page.

### Manual test

1. Open browser, navigate to DHIS2 server root.
2. Observe the URL is: `HOST/dhis-web-login`
3. Observe login is working,
